### DISCUSSION TITLE
안녕하세요, 작성하신 포스팅 보고 인사드리러 찾아왔습니다.

### DISCUSSION THREAD

**Discussion by ghost** [learningssik/pull-request#87] [Q&A] [answered]
discussions 탭은 자주 활용하지 않았었는데 이번 계기로 하나 더 알게 되었어요. issues 탭만 활용했던..🐓🐓🐓🐓💨
더 나은 커뮤니티 문화를 위해 힘쓰시는 모습 멋있고 감사드려요. 🙆🏻‍♂️

채택 관련해 답변 공간 열어 주시면 감사드려요, 안내해 주셔서 고맙습니다! 🤩

**Comment by learningssik** [comment, marked as answer]
안녕하세요 찾아주셔서 감사합니다!!
#90 #91 에 답변 남겨주시면 채택드릴게요~
